test.rb

```diff
@@ -0,0 +1,29 @@
+$:.unshift 'lib'
+require 'active_hash'
+
+class Foo < ActiveHash::Base
+  field :name
+  field :age
+  field :stuff, :private => true
```

Comment:
as said above, not a fan of private stuff.

Looking forward to seeing this an actual test

Reply:
@kbrock well, like i said, it's a case of getting data from a 3rd part that you're trying to wrap that you don't want exposed. Do you have a better suggestion?

Reply:
Hey @djberg96 - I get your use case and it sounds good.

Your example, with a few tweaks, was able to hide the attribute that you want hidden.
Since active record seems to work that way, this is the way I'm leaning.

The diff above got `test.rb` working for me.